src/page_body/structure/blocks/page_block_figma_components.pr

```diff
@@ -1 +1,104 @@
-Test 1
\ No newline at end of file
+{[ const block = context /]}
+
+{[ let configuration = exportConfiguration() /]}
+
+{[ const brands = ds.allBrands() /]}
+{[ const components = ds.allDesignComponents(brands[0].id) /]}
+
+{* TODO: this is not propagated from BE *}
+{[ let numberOfColumns = 2 /]}
+{[ let variant = "Grid" /]}
+
+{[ if (variant === "Grid") ]}
+    <div class="figma-components" style="grid-template-columns: repeat({{ numberOfColumns }}, 1fr);">
```

Comment:
We have a naming convention for all block wrappers which takes care of unified spacing between blocks

So it should be:
```
<div class="content-block content-block--figma-components {{ getVariantClass(variant) }}" id="{{ getSearchIDString(block.id) }}" ... >
```
...which will result into:
```
<div class="content-block content-block--figma-components variant-grid" id="block-id" ... >
```

Nit: In other places that support columns, we also don't use `grid-template-columns` as inline style, but we usuall use classname with a number for it, e.g. `layout-columns-{{ numberOfColumns }}`. One of the reasons is that it helps to better target the responsive behavior, where we decrease number of columns based on the screen size.

Reply:
[6dbd463](https://github.com/Supernova-Studio/exporter-documentation/pull/167/commits/6dbd46389004133fc1529e0db4877b3fa7456de8)